src/pages/navFooter.js

```diff
@@ -32,10 +32,16 @@ const NavBar = function (props) {
     Records
   </button>;
 
-  const btn_logout = <button className="btn btn-default navbar-btn" onClick={() => {
-    navigate("/");
-    cookies.remove('logged', { path: "/" });
-  }}>Logout</button>;
+  const btn_userinfo = <button
+    className='btn btn-default navbar-btn'
+    onClick={() => { return navigate("/userinfo") }}>
+    JWT
+  </button>
+
+  const btn_logout = <a href={window.config.logout}>
+    <button className="btn btn-default navbar-btn" onClick={() => {
+      cookies.remove('logged', { path: "/" });
```

Comment:
Why do we have the `cookies.remove` here?  Is that cookie being maintained separately from the others?

Reply:
The `logged` cookie is for the buttons UI in the navigation bar. Since we cannot access to the `session` cookie, I need something to indicate the button change.